plugins/c8y_configuration_plugin/src/upload.rs

```diff
@@ -434,88 +441,14 @@ mod tests {
             config_type: "config_type".to_string(),
         };
 
-        let plugin_config = PluginConfig {
-            files: HashSet::from([FileEntry::new_with_path_and_type(
-                "/some/test/config".to_string(),
-                "config_type".to_string(),
-            )]),
-        };
-
-        tokio::spawn(async move {
-            let _ = handle_config_upload_request(
-                &plugin_config,
-                config_upload_request,
-                &mut mqtt_client,
-                &mut http_client,
-                "".into(),
-                tedge_device_id,
-                Path::new(""),
-            )
-            .await;
-        });
-
-        // Assert the c8y_UploadConfigFile operation transitioning from EXECUTING(501) to SUCCESSFUL(503) with the uploaded config URL
-        mqtt_tests::assert_received_all_expected(
-            &mut messages,
-            TEST_TIMEOUT_MS,
-            &[
-                "501,c8y_UploadConfigFile",
-                "503,c8y_UploadConfigFile,http://server/config/file/url",
-            ],
-        )
-        .await;
-
-        Ok(())
-    }
-
-    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-    #[serial_test::serial]
-    async fn test_handle_config_upload_request_child_device() -> anyhow::Result<()> {
```

Comment:
Why was this test removed? This was the request mapping test for the child device and the one above was for the parent tedge device.